lms/djangoapps/courseware/field_overrides.py

```diff
@@ -70,6 +69,12 @@ def wrap(cls, user, wrapped):
         any performance impact of this feature if no override providers are
         configured.
         """
+        # if ccx not enabled for this course, just return unwrapped
+        from courseware.courses import get_course_by_id
+        course = get_course_by_id(course_id)
+        if not course.enable_ccx:
+            return wrapped
```

Comment:
@cpennington I made a comment about passing the course on the ticket. From looking at the code in module_render.py, I can't tell how to get the instantiated course. any advice here? Thanks.